Lib/defconAppKit/controls/glyphCollectionView.py

```diff
@@ -253,9 +253,10 @@ def setMode(self, mode):
             self._glyphCellView.unsubscribeFromWindow()
         elif mode == "cell":
             documentView = self._glyphCellView
-            if placard is not None:
-                placard.button.set(0)
+#             if placard is not None:
+#                 placard.button.set(0)
```

Comment:
This line triggers this traceback:
```python
Traceback (most recent call last):
  File "/TestApp/dist/DefconAppKit Test.app/Contents/Resources/DefconAppKitTest.py", line 43, in readFromFile_ofType_
    window = self.vanillaWindowController = DefconAppKitTestDocumentWindow(font)
  File "/TestApp/dist/DefconAppKit Test.app/Contents/Resources/DefconAppKitTest.py", line 96, in __init__
    selectionCallback=self.glyphViewCollectionSelectionCallback, showModePlacard=False)
  File "defconAppKit/controls/glyphCollectionView.pyc", line 224, in __init__
  File "defconAppKit/controls/glyphCollectionView.pyc", line 259, in setMode
<type 'exceptions.AttributeError'>: 'Group' object has no attribute 'button'
```
Commenting it does not seem to affect the functionality, but I'm not sure if it causes any side effects.